content/releases/prestashop80.md

```diff
@@ -0,0 +1,609 @@
+                            </p>
+                        </div>
+                        <div class="col-md-6">
+                            <p>
+                                <strong>Find out exactly what went wrong if the installation fails.</strong> If debug mode is on, a full stack trace will be displayed after something goes wrong during the installation process. This will definitely help developers pinpoint the source of the problem.
+                            </p>
+                            <p>
+                                <strong>Extend the installation process.</strong> Modules now include a callback <code>postInstall()</code>, allowing them to execute code after the shop’s installation is finished. This provides the developers with new possibilities, like add extra steps to the installation, preload assets, or preprocess data.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-md-6">
+                            <p>
+                                <a href="/releases/images/ps8/dkim.jpeg">
+                                    <img loading="lazy" class="image-with-shadow" src="/releases/images/ps8/dkim.jpeg">
+                                </a>
+                            </p>
+                        </div>
+                        <div class="col-md-6">
+                            <p>
+                                <a href="/releases/images/ps8/installer_errors.jpeg">
+                                    <img loading="lazy" class="image-with-shadow" src="/releases/images/ps8/installer_errors.jpeg">
+                                </a>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <section class="section" id="webservice">
+                <div class="container">
+                    <div class="row">
+                        <div class="col-md-4">
+                            <h2 class="section-title">Webservice</h2>
+                            <p>
+                                <strong>The status and URL to your shop’s webservice is now displayed</strong> at the top of the page in <em>Advanced parameters > Webservice</em>. This should be useful to new users and help them troubleshoot common problems.
+                            </p>
+                            <p><strong>*It is now possible to perform partial updates on webservice endpoints using the PATCH method.</strong> This allows integrations to update only part of an entity instead of all the fields at once. <a href="https://github.com/PrestaShop/PrestaShop/pull/27952">Learn more about this change here</a>.
```

Comment:
```suggestion
                            <p><strong>It is now possible to perform partial updates on webservice endpoints using the PATCH method.</strong> This allows integrations to update only part of an entity instead of all the fields at once. <a href="https://github.com/PrestaShop/PrestaShop/pull/27952">Learn more about this change here</a>.
```
Not visible in the preview, but there's a "*" at the beginning of the phrase that I removed